src/index.js

```diff
@@ -57,6 +57,14 @@ class LegacyUrls {
   servicesManifest() {
     return 'https://references.taskcluster.net/manifest.json';
   }
+
+  /**
+   * Returns a link to the task group or the task in Task Inspector
+   */
+  taskInspector(taskGroupId, taskId) {
+    let taskDetails = taskId ? `/tasks/${taskId}/runs/0/logs/public/logs/live.log` : '';
+    return 'https://tools.taskcluster.net/groups/' + taskGroupId + taskDetails;
```

Comment:
I prefer:
```
return ui('groups/' + taskGroupId + taskDetails)
```
since it doesn't duplicate the logic of the `ui` method, and if that method is changed, this method would pick up the changes.